libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -758,6 +791,17 @@ where
         }
     }
 
+    /// Adds outbound connection to matrix life cycle after it has been authorized
+    /// these connections are cannot be reconnected when dropped or lost.
```

Comment:
"... after it has been authorized these connections" doesn't parse.  I can't tell if punctuation is missing or if words need to be added (or removed).  Also, "it" is unclear - grammatically, it refers to the matrix life cycle, but I don't think that's what you meant. 

Here's my guess at the wording and punctuation changes that are needed:

"Adds the(?) [[or an?]] outbound connection to the(?) matrix life cycle. After a connection has been authorized, it cannot be reconnected when dropped or lost."